HARK/ConsumptionSaving/ConsIndShockModel.py

```diff
@@ -2035,6 +2076,8 @@ def check_conditions(self, verbose=None):
         "T_retire": 0,  # Period of retirement (0 --> no retirement)
         "vFuncBool": False,  # Whether to calculate the value function during solution
         "CubicBool": False,  # Use cubic spline interpolation when True, linear interpolation when False
+        "ntrl_msr": False,      # Use permanent income neutral measure during simulations when True.
```

Comment:
would it be possible to have this be a more legible parameter name?
like `neutral_measure` ?